tests/test_integration_org_unit.py

```diff
@@ -847,28 +904,28 @@ def test_rename_org_unit_early(self):
         self.assertRequestResponse(
             '/service/ou/{}/details/org_unit'.format(org_unit_uuid),
             [{
-                'name': 'Filosofisk Institut II',
+                'name': 'Whatever',
                 'org': {
                     'name': 'Aarhus Universitet',
                     'user_key': 'AU',
                     'uuid': '456362c4-0ee4-4e5e-a72c-751239745e62',
                 },
                 'org_unit_type': {
                     'example': None,
-                    'name': 'Institut',
+                    'name': 'Fakultet',
                     'scope': None,
-                    'user_key': 'inst',
-                    'uuid': 'ca76a441-6226-404f-88a9-31e02e420e52',
+                    'user_key': 'fak',
+                    'uuid': '4311e351-6a3c-4e7e-ae60-8a3b2938fbd6',
                 },
                 'parent': {
-                    'name': 'Humanistisk fakultet',
-                    'user_key': 'hum',
-                    'uuid': '9d07123e-47ac-4a9a-88c8-da82e3a4bc9e',
+                    'name': 'Overordnet Enhed',
+                    'user_key': 'root',
+                    'uuid': '2874e1dc-85e6-4269-823a-e1125484dfd3',
                 },
-                'user_key': 'fil',
+                'user_key': 'samf',
```

Comment:
Shouldn't we, eventually, validate the unit keys are unique? If so, this'll conflict with the preexisting unit.

Reply:
We should, eventually. The unit keys are randomly generated for now, with no way for the user to affect it, so I think we should cross this bridge when we get to it.